pkg/chart/testdata/ill/templates/wordpress.jinja

```diff
@@ -1,6 +1,6 @@
 #helm:generate dm_template
 {% set PROPERTIES = properties or {} %}
-{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-prod' %}
+{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-testing' %}
```

Comment:
This change and the others like it do not seem to affect any other content. Why do we have to revert them?